<launch>
  <!--arg name="pose_topic"      value="/mavros/local_position/pose" -->
  <arg name="pose_topic"      value="/ekf_node/ekf_pose"/>
  <!--arg name="pose_topic"      value="/orb_slam3/map_pose"-->

  <!-- Route Planner node -->
  <node pkg="control" name="route_planner" type="route_planner.py" output="screen">
    <remap from="/pose_topic" to="$(arg pose_topic)"/>

    <param name="Z_tag"                     value="0.70" />
    <param name="distance_threshold"        value="0.1" />
  </node>

  <!-- main node -->
  <node pkg="control" name="velocity_control" type="velocity_control.py" output="screen">
    <remap from="/pose_topic" to="$(arg pose_topic)"/>
    <param name="Z_tag"     value="0.70" />
  </node>

</launch>
